armi/reactor/converters/uniformMesh.py

```diff
@@ -197,6 +197,9 @@ def initNewReactor(sourceReactor, cs):
         coreDesign.construct(cs, bp, newReactor, loadAssems=False)
         newReactor.core.lib = sourceReactor.core.lib
         newReactor.core.setPitchUniform(sourceReactor.core.getAssemblyPitch())
+        newReactor.o = (
+            sourceReactor.o
+        )  # This is needed later for geometry transformation
```

Comment:
This is fine as a quick fix.

Seems like the real problem is that a geometry converter (or downstream repo?) is still depending on `r.o`.